verify/cert.go

```diff
@@ -166,3 +190,47 @@ func (cv *CertVerifier) retrieveBatchMetadataHash(ctx context.Context, batchID u
 	}
 	return onchainHash, nil
 }
+
+// getQuorumParametersAtLatestBlock fetches the required quorums and quorum adversary thresholds
+// from the EigenDAServiceManager contract at the latest block.
+// We then cache these parameters and use them in the Verifier to verify the certificates.
+//
+// Note: this strategy (fetching once and caching) only works because these parameters are immutable.
+// They might be different in different environments (for eg on a devnet or testnet), but they are fixed on a given network.
```

Comment:
nit: "for eg" is redundant, "e.g." expands to "for example"

Reply:
[ec8d910](https://github.com/Layr-Labs/eigenda-proxy/pull/241/commits/ec8d910a2a3cc7777b37024fa6f8acd52dd4dca3)